scenes/game/local_game.gd

```diff
@@ -5560,6 +5837,154 @@ func strike_send_attack_to_discard_or_gauge(performing_player : Player, card):
 
 	return events
 
+func begin_extra_attack(events, performing_player : Player, card_id : int):
+	assert(active_strike)
+	# The player is mid-strike and playing an extra attack card.
+	# These are the steps of extra attack:
+	# * Pay its cost.
+	# * Put it into play, gaining any Armor, Guard, and passive effects on it.
+	# * Perform its Before effects, then check its Range.
+	# * If it hit, perform its Hit effects.
+	# * If it hit, deal damage equal to its printed Power.
+	# * Perform its After effects.
+	# * At the end of the Strike, if the additional attack hit, add it to your Gauge. Otherwise, discard it.
+	# * Note that bonuses and effects that apply to your attack (e.g., Continuous Boost effects such as "+2 Power") only apply to your initial attack unless they specify "Your attacks have [...]" (as the Graviton does).
```

Comment:
hopefully this isn't something we have to find and support for other boosts retroactively?

Reply:
I think we just have to do a survey of all the things a given character that has extra attacks can boost.